src/storage.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable max-len */
+export default class Store {
+  static getTask = () => {
+    let tasks;
+    if (localStorage.getItem('tasks') === null) {
+      tasks = [];
+    } else {
+      tasks = JSON.parse(localStorage.getItem('tasks'));
+    }
+    return tasks;
+  }
+
+  static addTask = (task) => {
+    const tasks = Store.getTask();
+    task.index = tasks.length + 1;
+    tasks.push(task);
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+  }
+
+  static removeTask = (target) => {
+    if (target.classList.contains('remover')) {
+      const tasks = Store.getTask();
+      // eslint-disable-next-line eqeqeq
+      const taskItem = tasks.find((item) => item.index == target.parentElement.parentElement.firstChild.id);
+      tasks.splice(tasks.indexOf(taskItem), 1);
+      if (tasks.length > 0) {
+        let indexer = 1;
+        tasks.forEach((elem) => {
+          elem.index = indexer;
+          indexer += 1;
+        });
+      }
+      localStorage.setItem('tasks', JSON.stringify(tasks));
+    }
+  }
```

Comment:
Kindly update your getting started section to include updated instructions for how to run your project. As this project makes use of webpack, instructions such as `npm install` and `npm run start` should be included as this is how to start your application
<img width="640" alt="image" src="https://user-images.githubusercontent.com/37116322/228332423-5a86d58a-4b5b-488e-bb53-223fc87fab35.png">